<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OAProgress</class>
 <widget class="QDialog" name="OAProgress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>191</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>341</width>
    <height>171</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>341</width>
    <height>191</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Processing...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="OAOffline.qrc">
    <normaloff>:/icons/mainIcon</normaloff>:/icons/mainIcon</iconset>
  </property>
  <widget class="QProgressBar" name="progress">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Last file:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="filename">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QLabel" name="total_num">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Done file </string>
    </property>
   </widget>
   <widget class="QLabel" name="current_num">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>of</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="abort_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Time left:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="time_left">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="OAOffline.qrc"/>
 </resources>
 <connections/>
</ui>
